vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -105,6 +105,7 @@ The following is a list of specifications for the bundle descriptor and its comp
 |`permissions`|[Permission[]](#permission-specification)|No| | List of permissions to grant to the microservice |
 |`roles`|String[]|No||Exposed security roles|
 |`env`|[EnvironmentVariable[]](#environmentvariables-specification)|No||Required environment variables|
+| |[Microservices Environment Variables](#microservices-environment-variables)|No||Named Entando specific MS env variables |
```

Comment:
Entando-specific (hyphenated because these terms work together to become a single modifier of MS)